plugins/nodejs.yaml

```diff
@@ -178,4 +179,10 @@ pipeline:
       notice: 3xx
       warning: 4xx
       error: 5xx
+
+  - id: level_parser
+    type: severity_parser
+    if: '$record.level != nil'
```

Comment:
Should this check if $record.status exists as well?

Reply:
I added a check to ensure status doesn't exist. Without this it will overwrite statusCode severity parse if level also existed. Good catch.